project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import Keys._
 
 object Dependencies {
   lazy val http4sVersion = "0.18.4"
-  lazy val specs2Version = "4.0.3"
+  lazy val specs2Version = "4.0.4"
```

Comment:
Is there anything preventing us to go straight to [4.3.4](https://search.maven.org/artifact/org.specs2/specs2-core_2.12/4.3.4/jar)?

Reply:
Ah, just realized it was a PR made by robot.

Reply:
@chuwy The bot has a conservative update strategy. If there is a newer version with a greater patch version but the same major and minor versions it opens a PR for that first. Only if the current version is the latest patch version in a major.minor series it will open PRs with the latest minor version. So after this PR is merged the bot will open another PR for the update to 4.3.4.

Reply:
That makes sense. I just didn't realize at first this PR was not made by human (which probably means he has passed Turing test). Awesome project by the way, thank you!